js/views/modals/Settings/Page.js

```diff
@@ -232,7 +232,10 @@ export default class extends baseVw {
               new SimpleMessage({
```

Comment:
This is because cropit is no longer upscaling the image, it's saving it at the size of the original (with the cropping applied). 

I saw the issue yesterday, I kind of thought it was good to leave it in as a nag, but you're right, it should only warn you when you do the original upload and/or save (if the cropped image is too small after cropping).